test/unit/org/apache/cassandra/db/marshal/AbstractTypeTest.java

```diff
@@ -460,6 +460,15 @@ public void nested()
         });
     }
 
+    @Test
+    @SuppressWarnings("rawtypes")
+    public void nestedDuration()
+    {
+        qt().withShrinkCycles(0).forAll(AbstractTypeGenerators.builder().withoutTypeKinds(PRIMITIVE, COUNTER).withPrimitives(DurationType.instance).build()).checkAssert(type -> {
+            assertThat(type.referencesDuration());
```

Comment:
I think you want `.isFalse()` at the end, this doesn't actually assert or check things